crates/bevy_math/src/rects/rect.rs

```diff
@@ -309,6 +309,30 @@ impl Rect {
         r
     }
 
+    /// Express the coordinates of this rectangle relative to a normalized [0..1] x [0..1] space
+    /// defined by another rectangle.
```

Comment:
```suggestion
    /// Build a new rectangle from this one with its coordinates expressed
    /// relative to `other` in a normalized ([0..1] x [0..1]) coordinate system.
```

Nitpicky suggestion for the docs -- trying to match the style of other methods on `Rect`. Feel free to skip / modify if this is less accurate.

Reply:
Good suggestion, I wasn't that happy with it myself.